### ISSUE TITLE
Selection Highlight Background is opaque, hiding the selectionBackground

### ISSUE THREAD

**Issue by Mershl** [Monokai/monokai-pro-vscode#57]
Hi,

The Selection Highlight Background is the background color shown for items that match the current selection. The attribute is currently set to `#545f62` (fully opaque). VSCode draws the selectionBackground first and the selectionHighlightBackground afterwards leading to **the actual selection being displayed with the exact same background as other items that match the current selection**, which makes it impossible to differentiate between them.

The VSCode documentation contains the following comment for the attribute "editor.selectionHighlightBackground":
`Color for regions with the same content as the selection. The color must not be opaque to not hide underlying decoration.`

Is this intended by the theme?

**Comment by MartinHarding** [comment]
It appears that the following colors are also incorrectly set as opaque:
```
editor.findMatchHighlightBackground
editor.hoverHighlightBackground 
editor.rangeHighlightBackground
editor.wordHighlightBackground
editor.wordHighlightStrongBackground
merge.currentContentBackground
merge.currentHeaderBackground
merge.incomingContentBackground
merge.incomingHeaderBackground
```

`merge.commonContentBackground` and `merge.commonHeaderBackground` are absent from the definition as well, though I'm not sure that's as big of a deal.

Fortunately it's pretty easy to override the colors until there's an official fix. Here's my best attempt at mimicking Sublime Text behavior (unfortunately there's no foreground color for find highlighting, so you can't match it perfectly) for `Monokai Pro (Filter Machine)`:
```
{
    "workbench.colorCustomizations": {
        "[Monokai Pro (Filter Machine)]": {
            "editor.findMatchBackground": "#ffed7277",
            "editor.findMatchHighlightBackground": "#ffed7222",
            "editor.hoverHighlightBackground": "#545f6244",
            "editor.selectionBackground": "#545f62",
            "editor.selectionHighlightBackground": "#545f6266"
        }
    }
}
```

**Comment by Monokai** [comment]
Fixed in 1.1.6! Thanks :)
